src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/special/preferences/PreferencesPageObject.java

```diff
@@ -128,6 +131,28 @@ public void connectFacebook(String email, String password) {
     waitForElementByElement(facebookDisconnect);
   }
 
+  public void Use_advanced_recent_changes_checkbox() {
```

Comment:
method name should start from lower case letter and should be without special chars. Instead you can capitalize next words im method.
i.e. instead of `Use_advanced_recent_changes_checkbox` use `useAdvancedRecentChangesCheckbox`